### DISCUSSION TITLE
How about adding Table of Contents to Readme file

### DISCUSSION THREAD

**Discussion by puneeth072003** [OWASP/wrongsecrets#587] [Ideas]
### What do you think about adding Table of Contents to readme file since it is very long and hard to navigate through😃

**Comment by commjoen** [comment]
Good point @puneeth072003 ! But before we start adding it, I think it is time to, with https://github.com/OWASP/wrongsecrets/pull/586 and the existing wiki to see where we should put what… whada ya think @bendehaan ?

**Comment by puneeth072003** [comment]
@commjoen  Ya sure, i was a bit busy..... i submitted my PR tho it has to undergo reviewing.
